contract_compassion/wizard/end_contract_wizard.py

```diff
@@ -32,8 +32,9 @@ class EndContractWizard(models.TransientModel):
     do_transfer = fields.Boolean('I want to transfer the child')
     transfer_country_id = fields.Many2one(
         'compassion.global.partner', 'Country')
+    has_new_hold = fields.Boolean(default=True)

```

Comment:
Can you as well change the default value of field `end_date`? `fields.Datetime.now()` is better than using `datetime` library.